<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMultiAttributeEditor</class>
 <widget class="QDialog" name="CMultiAttributeEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="AttributeValuesTree">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="AddButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Add.svg</normaloff>:/Icons/Add.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="RemoveButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Delete.svg</normaloff>:/Icons/Delete.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="UpButton">
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Up.svg</normaloff>:/Icons/Up.svg</iconset>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DownButton">
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="icon">
        <iconset resource="../iqtgui/iqtgui.qrc">
         <normaloff>:/Icons/Down.svg</normaloff>:/Icons/Down.svg</iconset>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../iqtgui/iqtgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CMultiAttributeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CMultiAttributeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
